soc/arm/st_stm32/stm32g4/Kconfig.defconfig.stm32g431rb

```diff
@@ -11,4 +11,8 @@ config SOC
 config NUM_IRQS
 	default 102
 
+config DMAMUX_STM32
```

Comment:
DMAMUX is available on whole G4 series, I think this could safely be selected in soc/arm/st_stm32/stm32g4/Kconfig.defconfig.series